pyaedt/edb_core/EDB_Data.py

```diff
@@ -1827,6 +1827,11 @@ def __init__(self, edb_padstackinstance, _pedb):
         self._edb_padstackinstance = edb_padstackinstance
         self._pedb = _pedb
 
+    @property
+    def pin(self):
+        """Return Edb object."""
```

Comment:
Should we mention padstack here?
```suggestion
        """Return Edb padstack object."""
```